### PR TITLE
feat(tooling): add --build_config filter to 'kythe decor' command

### PR DESCRIPTION
**Pull request by schroederc**

### CODE REVIEW

File: kythe/go/platform/tools/kzip/createcmd/createcmd.go [hunk 1]
```diff
@@ -40,10 +41,10 @@ type createCommand struct {
 	rules  vnameRules
 
 	uri          kytheURI
-	source       repeatedStringSet
-	inputs       repeatedStringSet
+	source       flagutil.StringSet
+	inputs       flagutil.StringSet
 	hasError     bool
-	argument     repeatedString
+	argument     flagutil.StringList
```
Comment (shahms) [inline kythe/go/platform/tools/kzip/createcmd/createcmd.go]: These are incompatible with the existing and intended semantics of these fields, particularly "argument".

While `source` and `inputs` may reasonably (but not necessarily) expected to not contain any embedded commas, the same cannot be said of `argument`.

File: kythe/go/util/flagutil/flagutil.go [hunk 2]
```diff
@@ -73,3 +75,68 @@ func UsageError(msg string) {
 func UsageErrorf(str string, vals ...interface{}) {
 	UsageError(fmt.Sprintf(str, vals...))
 }
+
+// StringList implements a flag.Value that accepts an sequence of values as a CSV.
+type StringList []string
+
+// Set implements part of the flag.Getter interface and will append a new value to the flag.
```
Comment (shahms) [inline kythe/go/util/flagutil/flagutil.go]: "append new values" (unlike repeatedStringList, which will always only append a single value).

File: kythe/go/util/flagutil/flagutil.go [hunk 3]
```diff
@@ -73,3 +75,68 @@ func UsageError(msg string) {
...
+	if f == nil {
+		return []string(nil)
+	}
+	return *f
+}
+
+// StringSet implements a flag.Value that accepts an set of values as a CSV.
+type StringSet stringset.Set
+
+// Set implements part of the flag.Getter interface and will append a new value to the flag.
```
Comment (shahms) [inline kythe/go/util/flagutil/flagutil.go]: ditto on the plural here

File: kythe/go/platform/tools/kzip/createcmd/createcmd.go (hunk 1 above)
Comment (schroederc) [reply kythe/go/platform/tools/kzip/createcmd/createcmd.go]: I reverted that flag.

**Pull request merged**